step-templates/file-system-rsync.json

```diff
@@ -0,0 +1,73 @@
+{
+    "Id": "d8dfa484-59ed-491f-a22a-3098d4c2a6be",
+    "Name": "File System - rsync (bash)",
+    "Description": "Run [rsync](http://manpages.org/rsync) on a target or worker.",
+    "ActionType": "Octopus.Script",
+    "Version": 1,
+    "CommunityActionTemplateId": null,
+    "Packages": [],
+    "Properties": {
+        "Octopus.Action.Script.ScriptSource": "Inline",
+        "Octopus.Action.Script.Syntax": "Bash",
+        "Octopus.Action.Script.ScriptBody": "rsyncPath=$(get_octopusvariable \"Rsync.Path\")\nrsyncOptions=$(get_octopusvariable \"Rsync.Options\")\nrsyncSource=$(get_octopusvariable \"Rsync.Source\")\nrsyncDestination=$(get_octopusvariable \"Rsync.Destination\")\nprintCommand=$(get_octopusvariable \"FirebaseDeploy.PrintCommand\")\n\nif [ ! -z \"$rsyncPath\" ] ; then\n   \tPATH=$rsyncPath:$PATH\nfi\n\nif [ \"$printCommand\" = \"True\" ] ; then\n    set -x\nfi\n\nrsync ${rsyncOptions:+ $rsyncOptions} ${rsyncSource:+ $rsyncSource} ${rsyncDestination:+ $rsyncDestination}"
```

Comment:
ScriptBody references parameter which isn't supplied, namely: `FirebaseDeploy.PrintCommand`. Will fix up :+1:

Reply:
Done in commit a2201f7a